cli/command/registry.go

```diff
@@ -178,6 +180,9 @@ func PromptUserForCredentials(ctx context.Context, cli Cli, argUser, argPassword
 			}
 		}()
 
+		out := tui.NewOutput(cli.Out())
```

Comment:
```suggestion
		out := tui.NewOutput(cli.Err())
```

I think that should go to stderr?

Reply:
Because that's not really a part of a result, only a hint.

Reply:
ah yes! true